TrueSDK/Core/TCError.m

```diff
@@ -70,6 +70,9 @@ + (TCError *)errorWithCode:(TCTrueSDKErrorCode)code
         case TCTrueSDKErrorCodeInvalidName:
             errorDescription = @"Please provide a valid name";
             break;
+        case TCTrueSDKErrorCodeUniversalLinkFailed:
+            errorDescription = @"Universal Link failed";
```

Comment:
Can we get a better string here like, Cannot open Truecaller because of so and so

Reply:
changed to "Cannot open app because Universal Link failed"